### ISSUE TITLE
Fix github oauth

### ISSUE THREAD

**Issue by zol** [percolatestudio/atmosphere#18]
I changed the redirect url in github but for some reason now the oauth fails.

**Comment by zol** [comment]
This is really weird - the error is only happening on production, for some reason `Meteor.loginWithGithub` errors with:

```
e.LoginCancelledError {message: undefined, name: "Accounts.LoginCancelledError"}
```

**Comment by tmeasday** [comment]
Ok, there were a few things wrong here:
1. We didn't have a production github app setup. I created this one: https://github.com/organizations/percolatestudio/settings/applications/81562
   - I added the clientId/secret to ~/atmosphere/settings.json -- was this the right spot?
2. A problem with changing client id + secret, as fixed by https://github.com/percolatestudio/atmosphere/commit/dfa104e8a01af1438a72f0ff2f1f9d1e555dbc4f
3. The BASE_URL was misconfigured (to beta.atmosphere.com, we wish) in env.sh.
